schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(100);
```

Comment:
- [ ] @Salwa99 it's not good practice to edit already executed SQL command, you were supposed to use the  `ALTER TABLE` SQL command to add the new column, kindly add it correctly

Reply:
Thanks @danmainah , I'll fix this minor issue